test/commons/dom/inserted-into-focus-order.js

```diff
@@ -0,0 +1,211 @@
+describe('dom.insertedIntoFocusOrder', () => {
```

Comment:
Feel you could probably have combined some of these tests. It works though. Ok if you want to leave it like this.